### ISSUE TITLE
DarkRP Spawn Points/Jail Pos not saving after restart

### ISSUE THREAD

**Issue by josethe2nd** [FPtje/DarkRP#2784]
Description of the bug
DarkRP spawn points don't save after restart

How to make the bug happen

Why the developer of DarkRP is responsible for this issue:

Everything else saves, just not the darkrp spawn points and jail points. The server settings for FPP saves. I've seen many threads about this. I don't know how to fix this and I really don't understand how to either. I don't have any ERRORS or anything like that, no addons are causing this because I tried it without addons

**Comment by mikkelbiz** [comment]
I tested this with the latest version of DarkRP, and using both sqlite and mysql.

I set the position and closed my server application to simulate the method most hosting companies use to restart it, and it saved all 4 times.

1. Set jailpos (/setjailpos and /addjailpos) using sqlite and restart.
2. Set spawnpos (/setspawn cp) using sqlite and restart.
3. Set jailpos (/setjailpos and /addjailpos) using mysql and restart.
4. Set spawnpos (/setspawnpos cp) using mysql and restart.

All methods appeared to work, I would have to assume that its some kind of addon causing problems. Try renaming the addons folder to confirm. Then set the positions again.
